.storybook/preview.tsx

```diff
@@ -31,11 +32,13 @@ const preview: Preview = {
     Story => (
       <SiteProvider>
         <LocaleProvider>
-          <NotificationProvider viewportClassName="absolute top-0 left-0 list-none">
-            <div className={rootClasses}>
-              <Story />
-            </div>
-          </NotificationProvider>
+          <ThemeProvider>
```

Comment:
Why are you adding ThemeProvider on Storybook?

Reply:
if it's not aded icon switch doesn't work

Reply:
Please remove it. Why would you want the Theme Toggle button actually to toggle the Theme on a Storybook?

Components on Storybook are just to validate the design and interactions. Not things that are side-effects to the Component.

Reply:
Also, I would really appreciate it if you're more attentive to the history of files and PRs being merged as a **member of this project**.

The ThemeProvider was removed for numerous reasons (including as breaking the Storybook theme functionality, and other things, and mostly because we don't want theme switching from the real App within Storybook)

Reply:
I had removed my bad idea sorry 😅. I had try to make something cool but ...